src/explorer.ts

```diff
@@ -20,13 +20,28 @@ export function createKubernetesResource(kind: kuberesources.ResourceKind, id: s
 }
 
 function getIconForPodStatus(status: string): vscode.Uri {
-    if (status === "running" || status === "completed") {
+    if (status === "Running" || status === "Completed") {
```

Comment:
I'd suggest using a case-insensitive check so we don't have problems with possible different casings when the function is used from new places.

Reply:
Sounds good. Will change this.